sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -99,6 +103,16 @@ private[sql] object JDBCRelation extends Logging {
     }
     ans.toArray
   }
+
+  def getCurrentValue(columnType: Int, value: Long): String = {
```

Comment:
Probably, you can use `DateTimeUtils` to convert `currnetValue` to timestamp/date.